### ISSUE TITLE
"featured_image" behaves strangely if a URL link is in the description before the "more" separator.

### ISSUE THREAD

**Issue by zo0o0ot** [budparr/gohugo-theme-ananke#50]
I was confused by the size of my "featured_image" images changing to much smaller size on my index page seemingly at random.

I then figured out that my latest change introduced description text that included URL links in this format:

`_The morning of a Packer Game - [The NFC Championship in Green Bay before Super Bowl XXXI](http://www.pro-football-reference.com/boxscores/199701120gnb.htm)._`

If I include text like this before the `<!--more-->` separator, things get wonky.  If I include only text before the `<!--more-->`, everything returns to normal.  Right now, that's my workaround, but it'd be nice if this was fixed.

**Comment by budparr** [comment]
@zo0o0ot Are you talking about the `description` in front matter? If I put that copy in the front matter I see that it doesn't render as HTML, but it doesn't change the image. Just want to make sure we get the right solution for you.

**Comment by zo0o0ot** [comment]
I'm referring to the behavior of the main page of the site if there are posts with links in their description.

I actually had issues with the images resizing in unexpected ways if I had text with a link in the first portion of the post, before the `<!--more-->` separator.  If the links were after the `<!--more-->` separator, everything seemed fine.

On my site, I fixed the issue with this commit: https://github.com/sermonarchives/blog/commit/cc3dcaf13b4654f60fac56edfa46c7b33cd3b5a5

This was a sidestep of the main issue, I just moved all links out of the descriptions.
